bootstrap/examples/kubeflow_toolbox.yaml

```diff
@@ -0,0 +1,66 @@
+# Deploy kubeflow to namespace "kubeflow"
+
+# TODO(inc0): Create rbac role bindings
+---
+apiVersion: v1
+kind: Namespace
+metadata:
+  name: kubeflow
+
+# Headless service because StatefulSet requires it
+---
+kind: Service
+apiVersion: v1
+metadata:
+  name: kubeflow-toolbox
+  namespace: kubeflow
+spec:
+  clusterIP: "None"
+
+---
+apiVersion: v1
```

Comment:
@inc0 i ran into problems of a missing PV. I removed PersistentVolumeClaim and changed StatefulSet below. See https://cloud.google.com/kubernetes-engine/docs/how-to/stateful-apps

Reply:
PV is there for resiliency of app - this is meant to be long running pod that should survive restart of node - hence need for PV